docs/reference/search/rank-eval.asciidoc

```diff
@@ -7,9 +7,9 @@ list or manually rated documents, the `_rank_eval` endpoint calculates and
 returns typical information retrieval metrics like _mean reciprocal rank_,
 _precision_ or _discounted cumulative gain_.
 
-experimental[The ranking evaluation API is new and may change in non-backwards compatible ways in the future,
-even on minor versions updates.]
+experimental[The ranking evaluation API is new and may change in non-backwards compatible ways in the future, even on minor versions updates.]
```

Comment:
I missed that in the first review. I don't think you need to add anything to `experimental[]`, the message is auto generated with the appropriate content.

Reply:
There already was an experimental tag on master but it wasn't working as expected since I had a line break in the text. I added the custom text since the auto generated message reads `"This functionality is experimental and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but experimental features are not subject to the support SLA of official GA features."` which is fine, but I wanted to include the notion about changes in minor version updates since its likely we will slighty change the DSL a bit in the beginning.